kolibri/core/assets/src/vue/content-renderer/index.vue

```diff
@@ -174,9 +178,26 @@
           Object.assign(options, this.currentViewClass);
           // Instantiate the Vue instance directly using the Kolibri Vue constructor.
           this.contentView = new this.Kolibri.lib.vue(options); // eslint-disable-line new-cap
+          this.contentView.$on('startTracking', this.startTracking);
```

Comment:
We should wrap the actions themselves in methods that we then bind the listeners too, this will allow you to avoid having to pass in the `this.Kolibri` argument when you fire the events in the content renderers.